templates/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+genrule(
+    name = "templates-tarball",
+    srcs = glob([
+        "skeleton/**",
+        "quickstart-java/**",
+    ]) + ["//docs:quickstart-java.tar.gz"],
+    outs = ["templates-tarball.tar.gz"],
+    cmd = """
+        SRC=templates
+        OUT=templates-tarball
+
+        # skeleton template
+        mkdir -p $$OUT/skeleton
+        cp -rL $$SRC/skeleton/* $$OUT/skeleton/
+
+        # quickstart-java template
+        # right now, uses the preexisting quickstart-java rule and replaces the da.yaml template with a daml.yaml template
+        # in the future, move everything into //templates/quickstart-java and avoid untar, rm here
```

Comment:
Can we have a ticket for that? Or are you planning to do that as a fairly immediate follow up? Seems like a very valuable cleanup.

Reply:
Should be part of the da-assistant deprecation cleanup. I will create a ticket.

Reply:
Moving the quickstart out of `//docs` would require you to copy it back during the docs build to make the `literalinclude` directives work. You have to do the extra work in one of the places so I don't see a strong case for moving it.

Reply:
I see. It makes sense to me to put all the templates in one place (including the quickstart-java template), but that does sound a bit more involved. The da-assistant deprecation cleanup would still involve adding `daml.yaml.template` and removing `da-skeleton.yaml` from the template, whereven it is.